### PR TITLE
[MINI-5815] Support multiple emails in contacts

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
Added `allEmailList` field to support multiple emails for contacts.
Screen Shot 2022-12-21 at 1 55 48 PM

## Links
MINI-5815

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt [hunk 1]
```diff
@@ -7,5 +7,6 @@ import androidx.annotation.Keep
 data class Contact(
     val id: String,
     var name: String?,
-    var email: String?
+    var email: String?,
+    var allEmailList: List<String> = emptyList()
```
Comment (Climbatize) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt:11]: why empty empty list and not `List<String>?`

**Review by Climbatize — APPROVED** [review]

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt (hunk 1 above)
Comment (munir-rakuten) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt:11]: @Climbatize in that way hostapp it's optional, so host app can ignore it, is it a mandatory field?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - **Feature:** Added `NativeEventType.MINIAPP_RECEIVE_JSON_INFO` to provide `Universal Bridge` for receiving messages from the HostApp to MiniApps.
 - **Feature:** Added `sendJsonToMiniApp` in `MiniAppView` and `MiniAppViewImpl` for receiving messages from the HostApp to MiniApps.
 - **Fix:** Prevent file creation with zero byte with appending a single dot(.) at the end of file's name while downloading a file using `MiniAppFileDownloader` in Android 10.
+- **Update:** Added `allEmailList` field to support multiple emails for contacts.
```
Comment (khairul-alam-licon) [inline CHANGELOG.md:9]:
```suggestion
- **Update:** Added `allEmailList` field in `Contact` to support multiple emails of a specific contact.
```

**Review by khairul-alam-licon — APPROVED** [review]
LGTM

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt (hunk 1 above)
Comment (Climbatize) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt]: got it! Could have been `var allEmailList: List<String>? = null` then

**Pull request merged**